cypress/integration/e2e/management-tab/edit-worker.spec.ts

```diff
@@ -35,6 +35,42 @@ describe("Tab management", () => {
         });
       });
     });
+
+    describe("Editing the time", () => {
```

Comment:
`"Editing the time"` to jest wewnątrz bloku `"Creating worker"` a tutaj chcemy sprawdzić edycję etatu już istniejącego pracownika.

Reply:
Nie jest